features/guidance/staggered-intersections.feature

```diff
@@ -0,0 +1,72 @@
+@routing  @guidance @staggered-intersections
+Feature: Staggered Intersections
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 1 meters
+        # Note the one meter grid size: staggered intersections make zig-zags of a couple of meters only
+
+    # https://www.openstreetmap.org/#map=19/39.26022/-84.25144
+    Scenario: Staggered Intersection: Oak St, Cedar Dr
+        Given the node map
+            |   |   | j |   |   |
+            | a | b | c |   |   |
+            |   |   | d |   |   |
+            |   |   | e | f | g |
+            |   |   | h |   |   |
+            |   |   | i |   |   |
+
+        And the ways
+            | nodes  | highway     | name     |
+            | abc    | residential | Oak St   |
+            | efg    | residential | Oak St   |
```

Comment:
Can you add a negative test with `efg` not having the `Oak St` name? Like:

```
|abc|Oak Str|
|efg|Elm Str|
|jcdehi|Cedar Dr|
```

In that case the expected behavior would still be right-left/left-right turns, right?